src/spoke/libraries/UserPositionDebt.sol

```diff
@@ -94,13 +121,29 @@ library UserPositionDebt {
     return userPosition.getDebt(hub.getAssetDrawnIndex(assetId));
   }
 
-  /// @return The user's drawn debt.
+  /// @return The user's drawn debt, expressed in asset units.
   /// @return The user's premium debt, expressed in asset units and scaled by RAY.
   function getDebt(
     ISpoke.UserPosition storage userPosition,
     uint256 drawnIndex
   ) internal view returns (uint256, uint256) {
-    uint256 premiumDebtRay = userPosition.calculatePremiumRay(drawnIndex);
+    uint256 premiumDebtRay = userPosition._calculatePremiumRay(drawnIndex);
     return (userPosition.drawnShares.rayMulUp(drawnIndex), premiumDebtRay);
   }
+
+  /// @dev Calculates the premium debt of a user position with full precision.
+  /// @param userPosition The user position.
+  /// @param drawnIndex The current drawn index.
+  /// @return The premium debt, expressed in asset units and scaled by RAY.
+  function _calculatePremiumRay(
```

Comment:
I'd remove `_`, since this is a library fn

Reply:
ah correct, thought it's used outside of lib as well. then let's make it private, wdyt?

Reply:
we could call it as `_calculatePremiumRay(userPosition, drawnIndex)`, right? at this point I think we should be able to make it private. I think this would make sense, as it's weird to see `userPosition._calculate`, no?